docs/workflows/genomic_characterization/theiaprok.md

```diff
@@ -1238,6 +1244,30 @@ The TheiaProk workflows automatically activate taxa-specific sub-workflows after
 
     **Shigella XDR prediction.** Please see the documentation section above for ResFinder for details regarding this taxa-specific analysis. 
 
+    ??? task "`StxTyper`: Identification and typing of Shiga toxin (Stx) genes ==_using the assembly file as input_=="
+        
+        StxTyper screens bacterial genome assemblies for shiga toxin genes and subtypes them into known subtypes and also looks for novel subtypes in cases where the detected sequences diverge from the reference sequences.
+        
+        Shiga toxin is the main virulence factor of Shiga-toxin-producing E. coli (STEC), though these genes are also found in Shigella species as well as some other genera more rarely, such as Klebsiella. [Please see this review paper that describes shiga toxins in great detail.](https://doi.org/10.3390/microorganisms12040687)
+
+        !!! tip "Running StxTyper via the TheiaProk workflows"
+            The TheiaProk workflow will automatically run `stxtyper` on all E. coli and Shigella spp. samples, but ==*the user can opt-in to running the tool on any sample by setting the optional input variable `call_stxtyper` to `true` when configuring the workflow.*==
+        
+        Generally, `stxtyper` looks for _stxA_ and _stxB_ subunits that compose a complete operon. The A subunit is longer (in amino acid length) than the B subunit. Stxtyper attempts to detect these, compare them to a database of known sequences, and type them based on amino acid composition.  There typing algorithm and rules defining how to type these genes & operons will be described more completely in a publication that will be available in the future.
+        
+        The `stxtyper_report` output TSV is provided in [this output format.](https://github.com/ncbi/stxtyper?tab=readme-ov-file#output)
```

Comment:
can you link this to a static version so that when updates happen the link doesn't break

Reply:
like this? `https://github.com/ncbi/stxtyper#output`

or do you mean just simply `https://github.com/ncbi/stxtyper` ?

Reply:
i meant maybe tag to a version or something in case they reorg their docs -- maybe the plain github readme would work

Reply:
mainly if they change it to be #output-file or something idk

Reply:
how about this, a link to the output section of the v1.0.24 tag: `https://github.com/ncbi/stxtyper/tree/v1.0.24?tab=readme-ov-file#output` ?

Reply:
perfect, thank works!

Reply:
done, thank you for the suggestion